src/dataService/utils.spec.ts

```diff
@@ -1,32 +1,82 @@
-import { expect } from 'chai';
-import { mysqlDateConvert, mysqlTimeString } from './utils';
+import {
+  mysqlDateConvert,
+  mysqlTimeString,
+  setDifference,
+  uniqueArray,
+} from './utils';
 
 const dateGood = new Date('2008-11-03 08:51:01');
 const dateNull = new Date('0000-00-00 00:00:00');
 
 describe('mysqlDateConvert', () => {
   it('valid date in, valid date out', () => {
     const result = mysqlDateConvert(dateGood);
-    expect(result).to.equal(dateGood);
+    expect(result).toEqual(dateGood);
   });
   it('invalid 0000-00-00 date in, null out', () => {
     const result = mysqlDateConvert(dateNull);
-    expect(result).to.equal(null);
+    expect(result).toBeNull();
   });
   it('invalid string in, null out', () => {
     const result = mysqlDateConvert('notadate');
-    expect(result).to.equal(null);
+    expect(result).toBeNull();
   });
   it('null in, null out', () => {
     const result = mysqlDateConvert(null);
-    expect(result).to.equal(null);
+    expect(result).toBeNull();
```

Comment:
Thanks for asking!  This is because I moved from using chai to jest. The most compelling reason is probably streamlining (we use jest already, and it works great, so why add chai?). I've also had trouble in the past with some of chai's deep/array assertions, and the jest-extended library provides a number of really nice assertions (like toIncludeSameMembers which I'm using below).